src/atoms/Button/Button.web.js

```diff
@@ -1,20 +1,454 @@
+      case 'secondary':
+        return `${makePxValue(0.125)} solid ${getColor(theme, `${color}.900`)}`;
+      case 'tertiary':
+        return '0px';
+      default:
+        return `${makePxValue(0.125)} solid ${getColor(theme, 'shade.960')}`;
+    }
+  },
+  focusBackgroundColor({ theme, variant, color, disabled }) {
+    if (disabled) {
+      return '';
+    }
+    switch (variant) {
+      case 'primary':
+        return getColor(theme, `${color}.800`);
+      case 'secondary':
+        return getColor(theme, `${color}.930`);
+      case 'tertiary':
+        return getColor(theme, `${color}.930`);
+      default:
+        return getColor(theme, `${color}.800`);
+    }
+  },
+  activeBorder({ theme, variant, disabled, color }) {
+    if (disabled) {
+      return '';
+    }
+    switch (variant) {
+      case 'primary':
+        return '0px';
+      case 'secondary':
+        return `${makePxValue(0.125)} solid ${getColor(theme, `${color}.970`)}`;
+      case 'tertiary':
+        return '0px';
+      default:
+        return '0px';
+    }
+  },
+  activeBackgroundColor({ theme, variant, color, disabled }) {
+    if (disabled) {
+      return '';
+    }
+    switch (variant) {
+      case 'primary':
+        return getColor(theme, `${color}.600`);
+      case 'secondary':
+        return getColor(theme, `${color}.950`);
+      case 'tertiary':
+        return getColor(theme, `${color}.950`);
+      default:
+        return getColor(theme, `${color}.600`);
+    }
+  },
+  hoverBorder({ theme, variant, disabled, color }) {
+    if (disabled) {
+      return '';
+    }
+    switch (variant) {
+      case 'primary':
+        return '0px';
+      case 'secondary':
+        return `${makePxValue(0.125)} solid ${getColor(theme, `${color}.970`)}`;
+      case 'tertiary':
+        return '0px';
+      default:
+        return '0px';
+    }
+  },
+  hoverBackgroundColor({ theme, variant, color, disabled }) {
+    if (disabled) {
+      return '';
+    }
+    switch (variant) {
+      case 'primary':
+        return getColor(theme, `${color}.700`);
+      case 'secondary':
+        return getColor(theme, `${color}.930`);
+      case 'tertiary':
+        return getColor(theme, `${color}.920`);
+      default:
+        return getColor(theme, `${color}.700`);
+    }
+  },
+  border({ variant, color, disabled, theme }) {
+    switch (variant) {
+      case 'primary':
+        return '0px';
+      case 'secondary':
+        if (disabled) {
+          return `${makePxValue(0.125)} solid ${getColor(theme, 'shade.930')}`;
+        }
+        return `${makePxValue(0.125)} solid ${getColor(theme, `${color}.970`)}`;
+      case 'tertiary':
+        return '0px';
+      default:
+        return '0px';
+    }
+  },
+  height({ size, theme }) {
+    switch (size) {
+      case 'xsmall':
+        return theme.spacings.xlarge;
+      case 'small':
+        return makePxValue(3.5); //TODO: not on scale of 8
```

Comment:
What's the issue here? How much is the height in absolute px?

Reply:
removed by confirming with figma

Reply:
@shubhnik Can you also please check and remove these comments from the Button.native.js as well? Thanks!